src/repository/sharing.rs

```diff
@@ -0,0 +1,152 @@
+use axum::async_trait;
+use mockall::automock;
+use rand::{distributions::Alphanumeric, Rng};
+use sqlx::PgPool;
+use tracing::{error, info, warn};
+
+use crate::domain::crud::CrudInt;
+
+pub enum ProjectSharingCreateError {
+    Unknown,
+}
+pub enum ProjectSharingUpdateError {
+    Unknown,
+}
```

Comment:
I don't require it to change, but mark as TODO

Reply:
ok